README.md

```diff
@@ -41,15 +41,15 @@ pytest
 This is an example of how to run the code where the targetstore is a file / in memory store.
 
 ```bash
-python ./pyTravHarv/**main**.py -cf ./tests/config/ -n base_test.yml -ts ./tests/inputs/63523.ttl -v
+python -m pytravharv -cf ./tests/config/ -n base_test.yml -m memory -o ./test.ttl -v -c ./tests/inputs/63523.ttl ./test.ttl
 ```
 
 #### example 2
 
 This is an example of how to run the code where the targetstore is a SPARQL endpoint.
 
 ```bash
-python ./pyTravHarv/**main**.py -cf ./tests/config/ -n base_test.yml -ts http://example.com/repo/id -v
+python -m pytravharv -cf ./tests/config/ -n base_test.yml -m uristore -o ./test.ttl -v -ts http://example.org http://example.org/statements
 ```
```

Comment:
cli switches not really intuitive
reads like a strange mix of concerns ?

Reply:
issue mentioned in #15